core/src/components/select/test/select.spec.tsx

```diff
@@ -125,3 +125,35 @@ describe('select: slot interactivity', () => {
     expect(divSpy).toHaveBeenCalled();
   });
 });
+
+describe('ion-select: required', () => {
+  it('should have a aria-required attribute as true in inner button', async () => {
+    const page = await newSpecPage({
+      components: [Select],
+      html: `
+        <ion-select required="true"></ion-select>
+      `,
+    });
+
+    const select = page.body.querySelector('ion-select')!;
+
+    const nativeButton = select.shadowRoot!.querySelector('button')!;
+
+    expect(nativeButton.getAttribute('aria-required')).toBe('true');
+  });
+
+  it('should not have a aria-required attribute as false in inner button', async () => {
```

Comment:
```suggestion
  it('the inner button should have the aria-required attribute set to false', async () => {
```

Reply:
The current version aligns with the testing names standards for Ionic. Please leave it as is.